spaghetto/README.md

```diff
@@ -46,6 +46,48 @@ package:
     #   2. the package, using a specified vesrion range
     - packageName: ">=1.1.1 <2.0.0"
 
+# optional
+  build:
+    # optional
+    psaOptions:
```

Comment:
I would not nest this under `psaOptions`, they can all be "build options"

Reply:
Oh good point. If we drop the "psa" idea and just see this as a build option, then it can go up one more level.